### PR TITLE
[4.0] Add container-fluid to com_template modals

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue # .

### Summary of Changes
Applies #14028 to 4.0-dev branch

### Testing Instructions
4.0-dev branch

### CODE REVIEW

**Comment by franz-wohlkoenig** [comment]
as Patchtester don't fetch Data unable to test.

File: administrator/components/com_templates/views/template/tmpl/default_modal_copy_body.php [hunk 1]
```diff
@@ -9,15 +9,19 @@
...
+	<div class="row-fluid">
+		<div class="col-md-12">
+			<div class="control-group">
+				<div class="control-label">
+					<label for="new_name" class="modalTooltip" title="<?php echo JHtml::_('tooltipText', 'COM_TEMPLATES_TEMPLATE_NEW_NAME_LABEL', 'COM_TEMPLATES_TEMPLATE_NEW_NAME_DESC'); ?>">
+						<?php echo JText::_('COM_TEMPLATES_TEMPLATE_NEW_NAME_LABEL'); ?>
+					</label>
+				</div>
+				<div class="controls">
+					<input class="input-xlarge" type="text" id="new_name" name="new_name"  />
```
Comment (yvesh) [inline administrator/components/com_templates/views/template/tmpl/default_modal_copy_body.php]: input-xlarge in bs4? And HTML 5 style ;) (! />)

File: administrator/components/com_templates/views/template/tmpl/default_modal_delete_body.php [hunk 2]
```diff
@@ -11,6 +11,8 @@
 ?>
 <div id="template-manager-delete" class="container-fluid">
 	<div class="row-fluid">
```
Comment (yvesh) [inline administrator/components/com_templates/views/template/tmpl/default_modal_delete_body.php]: There is no row-fluid in bs4 more ;) -> row

File: administrator/components/com_templates/views/template/tmpl/default_modal_file_body.php [hunk 3]
```diff
@@ -11,58 +11,60 @@
...
-					<input type="submit" value="<?php echo JText::_('COM_TEMPLATES_BUTTON_UPLOAD'); ?>" class="btn btn-primary" /><br>
-					<?php $cMax    = $this->state->get('params')->get('upload_limit'); ?>
-					<?php $maxSize = JHtml::_('number.bytes', JUtility::getMaxUploadSize($cMax . 'MB')); ?>
-					<?php echo JText::sprintf('JGLOBAL_MAXIMUM_UPLOAD_SIZE_LIMIT', $maxSize); ?>
-				</fieldset>
-			</form>
-			<?php if ($this->type != 'home') : ?>
-				<form method="post" action="<?php echo JRoute::_('index.php?option=com_templates&task=template.copyFile&id=' . $input->getInt('id') . '&file=' . $this->file); ?>" class="well" enctype="multipart/form-data">
+<div id="#template-manager-file" class="container-fluid">
+	<div class="row-fluid">
```
Comment (yvesh) [inline administrator/components/com_templates/views/template/tmpl/default_modal_file_body.php]: -> row

File: administrator/components/com_templates/views/template/tmpl/default_modal_folder_body.php [hunk 4]
```diff
@@ -11,22 +11,24 @@
...
-					<?php echo JHtml::_('form.token'); ?>
-					<input type="submit" value="<?php echo JText::_('COM_TEMPLATES_BUTTON_CREATE'); ?>" class="btn btn-primary" />
-				</fieldset>
-			</form>
-		</div>
-		<div class="col-md-6 column-left">
-			<?php echo $this->loadTemplate('folders'); ?>
-			<hr class="hr-condensed" />
+<div id="#template-manager-folder" class="container-fluid">
+	<div class="row-fluid">
```
Comment (yvesh) [inline administrator/components/com_templates/views/template/tmpl/default_modal_folder_body.php]: -> row

File: administrator/components/com_templates/views/template/tmpl/default_modal_rename_body.php [hunk 5]
```diff
@@ -10,17 +10,21 @@
...
+		<div class="col-md-12">
+			<div class="control-group">
+				<div class="control-label">
+					<label for="new_name" class="modalTooltip" title="<?php echo JHtml::_('tooltipText', JText::_('COM_TEMPLATES_NEW_FILE_NAME')); ?>">
+						<?php echo JText::_('COM_TEMPLATES_NEW_FILE_NAME')?>
+					</label>
+				</div>
+				<div class="controls">
+					<div class="input-append">
+						<input class="input-xlarge" type="text" name="new_name" required />
```
Comment (yvesh) [inline administrator/components/com_templates/views/template/tmpl/default_modal_rename_body.php]: form-control

File: administrator/components/com_templates/views/template/tmpl/default_modal_resize_body.php [hunk 6]
```diff
@@ -9,25 +9,29 @@
...
-	<div class="control-group">
-		<div class="control-label">
-			<label for="width" class="modalTooltip" title="<?php echo JHtml::_('tooltipText', 'COM_TEMPLATES_IMAGE_WIDTH'); ?>">
-				<?php echo JText::_('COM_TEMPLATES_IMAGE_WIDTH')?>
-			</label>
-		</div>
-		<div class="controls">
-			<input class="input-xlarge" type="number" name="width" placeholder="<?php echo $this->image['width']; ?> px" required />
+<div id="template-manager-resize" class="container-fluid">
+	<div class="row-fluid">
```
Comment (yvesh) [inline administrator/components/com_templates/views/template/tmpl/default_modal_resize_body.php]: row

File: administrator/components/com_templates/views/template/tmpl/default_modal_resize_body.php [hunk 7]
```diff
@@ -9,25 +9,29 @@
...
+		<div class="col-md-12">
+			<div class="control-group">
+				<div class="control-label">
+					<label for="height" class="modalTooltip" title="<?php echo JHtml::_('tooltipText', 'COM_TEMPLATES_IMAGE_HEIGHT'); ?>">
+						<?php echo JText::_('COM_TEMPLATES_IMAGE_HEIGHT')?>
+					</label>
+				</div>
+				<div class="controls">
+					<input class="input-xlarge" type="number" name="height" placeholder="<?php echo $this->image['height']; ?> px" required />
```
Comment (yvesh) [inline administrator/components/com_templates/views/template/tmpl/default_modal_resize_body.php]: form-control

File: administrator/components/com_templates/views/template/tmpl/default_modal_resize_body.php [hunk 8]
```diff
@@ -9,25 +9,29 @@
...
+				</div>
+			</div>
+			<div class="control-group">
+				<div class="control-label">
+					<label for="width" class="modalTooltip" title="<?php echo JHtml::_('tooltipText', 'COM_TEMPLATES_IMAGE_WIDTH'); ?>">
+						<?php echo JText::_('COM_TEMPLATES_IMAGE_WIDTH')?>
+					</label>
+				</div>
+				<div class="controls">
+					<input class="input-xlarge" type="number" name="width" placeholder="<?php echo $this->image['width']; ?> px" required />
```
Comment (yvesh) [inline administrator/components/com_templates/views/template/tmpl/default_modal_resize_body.php]: form-control

File: administrator/components/com_templates/views/template/tmpl/default_modal_copy_body.php (hunk 1 above)
Comment (C-Lodder) [inline administrator/components/com_templates/views/template/tmpl/default_modal_copy_body.php]: Don't worry too much about the `/ >` as I'm fixing these in staging which will then be merged.

**Comment by ciar4n** [comment]
Thank you @yvesh

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on 2672717c1c92bb42c4bd99aa77893b10349bca06<br /><br />working fine now<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14078">issues.joomla.org/tracker/joomla-cms/14078</a>.</sub>

**Comment by yvesh** [comment]
I have tested this item :white_check_mark: successfully on 2672717c1c92bb42c4bd99aa77893b10349bca06<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14078">issues.joomla.org/tracker/joomla-cms/14078</a>.</sub>

**Pull request merged**